Added a file explaining the built-in function head() from the pandas'… (#4503)
Codecademy/docs: content/pandas/concepts/built-in-functions/terms/head/head.md

---
Title: '.head()'

> Description: 'Returns the first 5 rows of a dataset provided before.'

Reviewer: ```suggestion
Description: 'Returns the specified number of rows from the start of the dataset or DataFrame provided.'
```

Subjects:
  - 'Data Science'
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Display'
  - 'Pandas'
CatalogContent:
  - 'learn-python-3'
  - 'paths/data-science-foundations'
---

> Using the buil-in function `.head()` will display all the columns of a dataset and the first 5 rows of information for each column. Also you can pass an integer as an argument in order to see that number of rows displayed. 

Reviewer: ```suggestion
The **`.head()`** function from the pandas library returns all columns of the specified number of rows from the start of the dataset or DataFrame provided.
```

## Syntax 

```pseudo

> pd.head(10)

Reviewer: ```suggestion
pd.head(rows)
```

```

> The parameter "10" infomrs us that the rows displayed will change from 5 (default) to 10. 

Reviewer: ```suggestion
- `rows`: It takes an `int` value that stands for the number of rows to be returned.

> **Note:** By default `rows` takes the value 5 if a value is not specified.
```

> ## Example

Reviewer: ```suggestion
## Example

```
Add a extra line here.

```py

> import pandas as pd
> 
> document = pd.read_csv("csv_file.csv")
> 
> document.head()

Reviewer: ```suggestion
import pandas as pd

document = pd.DataFrame({"fruits": ['mango', 'blackberry', 'watermelon', 'oranges', 'apple', 'grapefruit'], "season": ['spring', 'summer', 'summer', 'winter', 'fall', 'winter']})

print(document.head(2))
```
This example makes it clearer.

```

> The code can be changed by using another kind of file such as an Excel file (with the extension .xlsx), and the parameter in `.head()` can be omitted . This code would result on the following output: 

Reviewer: ```suggestion
This code would result in the following output: 
```

```shell

>   column 1 column 2 column 3 column 4 column 5
> 0        A        B        C        D       E
> 1        F        G        H        I       J
> 2        K        L        M        N       O
> 3        P        Q        R        S       T
> 4        U        V        W        X       Y
> 

Reviewer: ```suggestion
       fruits  season
0       mango  spring
1  blackberry  summer
```
This is the corresponding output for the new code.

```

> The letters in the columns will be replaced with the information from the document provided when reading the file. 

Reviewer: ```suggestion
> **Note:** The `.head()` function can used with a `.csv` file. 
```
